met-web/src/App.tsx

```diff
@@ -60,16 +62,21 @@ const App = () => {
 
     return (
         <Router>
-            <Box sx={{ display: 'flex' }}>
-                <LoggedInHeader drawerWidth={drawerWidth} />
-                <Notification />
-                <NotificationModal />
-                <Box component="main" sx={{ flexGrow: 1, width: `calc(100% - ${drawerWidth}px)`, marginTop: '17px' }}>
-                    <Toolbar />
-                    <AuthenticatedRoutes />
-                    <FeedbackModal />
+            <DndProvider backend={HTML5Backend}>
```

Comment:
if we needed providers can we add them with the other providers in the index.tsx file?